.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-https://github.com/nimblehq/ios-templates/issues/
+https://github.com/nimbl3/ios-templates/issues/
```

Comment:
I should be `nimblehq` :D

Reply:
I think the rebase made them disappear, I will recheck again.